packages/playground/src/dashboard/transfer_view.vue

```diff
@@ -37,8 +37,11 @@
                 :rules="[
                   validators.required('Transfer amount is required '),
                   validators.isNumeric('Amount should be a number.'),
-                  validators.min('Amount must be greater than 0', 0.00000000001),
-                  validators.max('Insufficient funds', freeBalance),
+                  validators.min('Amount must be greater than 0.', 0.001),
+                  validators.max('Insufficient funds.', freeBalance),
+                  validators.pattern('Amount can have 3 decimals only.', {
+                    pattern: /^\d+(\.\d{1,3})?$/,
```

Comment:
I think we can use `isDecimal` instead of pattern

Reply:
Fixed.